docs/standard/containerized-lifecycle-architecture/docker-devops-workflow/docker-application-outer-loop-devops-workflow.md

```diff
@@ -43,191 +43,169 @@ When using Docker for the deployment, the "final artifacts" to be deployed are D
 
-![](./media/image11.png)
+![Browser view of the task catalog in Azure DevOps, showing the Docker deploy task.](./media/image10.png)
 
-Figure 5-11: Docker Deploy task definition deploying to Azure Container Service DC/OS
+**Figure 5-10**. Adding the Docker Deploy task to your Environment RM
 
-**More info** To read more about the CD pipeline with Azure DevOps Services and Docker, visit the following sites:
+Figure 5-11 demonstrates how you can edit the Docker Deploy task and specify the Target Type (Azure Container Service DC/OS, in this case), your Docker Compose File, and the Docker Registry connection (like Azure Container Registry or Docker Hub). This is the task that will retrieve your ready-to-use custom Docker images to be deployed as containers in the cluster.
 
-Azure DevOps Services extension for Docker and Azure Container Service: [https://aka.ms/\
-vstsdockerextension](https://aka.ms/vstsdockerextension)
+![Browser view of Azure DevOps, deploy to orchestrator task definition.](./media/image11.png)
 
-Azure Container Service: <https://aka.ms/azurecontainerservice>
+**Figure 5-11**. Docker Deploy task definition deploying to Azure Container Service DC/OS
 
-Mesosphere DC/OS: <https://mesosphere.com/product/>
+> [!INFORMATION]
+> To read more about the CD pipeline with Azure DevOps Services and Docker, visit <https://azure.microsoft.com/services/devops/pipelines>
 
 ## Step 5: Run and manage
 
 Because running and managing applications at enterprise-production level is a major subject in and of itself, and due to the type of operations and people working at that level (IT operations) as well as the large scope of this area, we have devoted the entire next chapter to explaining it.
 
 ## Step 6: Monitor and diagnose
 
-This topic also is covered in the next chapter as part of the tasks that IT operations performs in production systems; however, is important to highlight that the insights obtained in this step must feed back to the development team so that the application is constantly improved. From that point of view, it is also part of DevOps, although the tasks and operations are usually performed by IT.
+This topic also is covered in the next chapter as part of the tasks that IT performs in production systems; however, is important to highlight that the insights obtained in this step must feed back to the development team so that the application is constantly improved. From that point of view, it is also part of DevOps, although the tasks and operations are commonly performed by IT.
```

Comment:
Topic is legacy terminology 

```suggestion
This article also is covered in the next chapter as part of the tasks that IT performs in production systems; however, is important to highlight that the insights obtained in this step must feed back to the development team so that the application is constantly improved. From that point of view, it is also part of DevOps, although the tasks and operations are commonly performed by IT.
```

Reply:
@mairaw I'm not sure about the change from topic to article, because it refers to the "Monitor(ing) and diagnose(ing)" topic or subject, and that's what's covered in another chapter.

Reply:
I agree; "topic" is correct in this context. I'll merge your PR now, @mvelosop.